.github/workflows/go.yml

```diff
@@ -39,9 +39,9 @@ jobs:
       - name: Release
         if: github.ref == 'refs/heads/master' # only run on master branch
         run: |
-          curl -LO https://releases.manifold.co/manifold-cli/$(MANIFOLD_VERSION)/manifold-cli_$(MANIFOLD_VERSION)_linux_amd64.tar.gz
+          curl -LO https://releases.manifold.co/manifold-cli/$MANIFOLD_VERSION/manifold-cli_$MANIFOLD_VERSION_linux_amd64.tar.gz
```

Comment:
`${MANIFOLD_VERSION}` should work better in GitHub Actions.

Also does that need to be in `env:` in the top-level of the job?

Reply:
Oh nevermind; I see it in `env` for this step so that’s fine